eng/Subsets.props

```diff
@@ -315,7 +303,7 @@
     <ProjectToBuild Include="$(CoreClrProjectRoot)nativeaot\**\src\*.csproj" Category="clr" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(_subset.Contains('+clr.nativeaotbuild+'))">
+  <ItemGroup Condition="$(_subset.Contains('+clr.nativeaotruntime+'))">
```

Comment:
Hmm. So we're adding the build integration into native runtime build, which is not very logical, but does keep the command line length smaller. I am ok with this for the time being, but once we merge `clr.aot`, we should delete this, so there should be a `TODO-LLVM` about that.